### DISCUSSION TITLE
How open the settings menu.

### DISCUSSION THREAD

**Discussion by milfer322** [valinet/ExplorerPatcher#232] [Q&A] [answered]
How can oppen the settings menu?
I have taskbar windows 11 style, with win+x cant open the settings menu.

Add capture.
![image](https://user-images.githubusercontent.com/17482316/139490298-87e13b4a-7c07-4124-901a-062557246905.png)
Dont have the button "Properties" for open the menu.

Thanks so much

**Comment by ExorcistF1** [comment, marked as answer]
run(win+R) rundll32.exe C:\Windows\dxgi.dll,ZZGUI

**Reply by yetisyny** [reply]
To make it easier to open settings for ExplorerPatcher, people can create a shortcut to "rundll32.exe C:\Windows\dxgi.dll,ZZGUI" by right-clicking the Desktop and creating a new shortcut, so they don’t have to open the Run dialog every time. Then rename the shortcut "ExplorerPatcher" or "ExplorerPatcher Settings" or something.

For the icon to use with the shortcut, I suggest changing this new shortcut’s icon to be one of the icons in C:\Windows\explorer.exe, the 16th icon, an icon of the Taskbar and Start menu that used to be used for Taskbar and Start menu settings in older versions of windows.

You can then copy/move the shortcut from the Desktop to the Start menu if you want and even assign a hotkey to it. Then you can easily open ExplorerPatcher settings at any time without having to remember the Win-X, Properties way of getting there, or if Win-X, Properties isn’t working for you. I don’t typically use Win-key combinations so this is what I do, have a shortcut for it in my Start menu.
